### PR TITLE
Swapped order of sentences

### PR DESCRIPTION
**Pull request by anenadic**
Swapped order of sentences to make sure self-learning mode is introduced and explained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,13 @@ and is in a good state to be reused and taught by others.
...
-standard instructor-led delivery or for self-learning with some guidance from instructors who can provide help 
-and answer questions (synchronously or asynchrounously) as learners go through the course. Initially, in sections 1-3 
-of the lesson, learners are working on a software project and going though exercises on their own. In sections 4 and 5,
-they are grouped and work in teams, as they would when collaborating on a software project involving several developers.
+The lesson is suitable for both instructor-led teaching or guided self-learning where instructors provide help 
+and answer questions (synchronously or asynchrounously) as learners go through the course on their own. Initially, in sections 1-3 
+of the lesson, learners are working on a software project and going though exercises individually. In sections 4 and 5,
+they are grouped and work in teams, as they would when collaborating on a team software project development.
```
Comment (bielsnohr) [inline README.md:33]: Sembr
```suggestion
The lesson is suitable for both instructor-led teaching or guided self-learning where instructors provide help 
and answer questions (synchronously or asynchrounously) as learners go through the course on their own. 
Initially, in sections 1-3 
of the lesson, learners are working on a software project and going though exercises individually.
In sections 4 and 5,
they are grouped and work in teams, as they would when collaborating on a team software project development.
```

File: README.md [hunk 2]
```diff
@@ -27,11 +27,13 @@ and is in a good state to be reused and taught by others.
...
+
+The lesson has 5 sections; each section can be delivered or worked through in self-learning mode over a half a day to a day 
+depending on the pace (we would recommend setting aside 1 day per section to make sure the delivery is not rushed).
```
Comment (bielsnohr) [inline README.md:36]: Sembr and punctuation changes

```suggestion
The lesson has 5 sections; 
each section can be delivered or worked through in self-learning mode over a half a day to a day, 
depending on the pace.
We would recommend setting aside one day per section to make sure the delivery is not rushed. 
```

**Review by bielsnohr — CHANGES_REQUESTED** [review]
LGTM. Just a few minor sembr changes.

**Review by anenadic — COMMENTED** [review]
Thank you for the comments @bielsnohr - if this is looking OK now do you mind approving the PR please so it can be merged?

**Review by bielsnohr — APPROVED** [review]
Made a few quick changes myself. All good now.

**Pull request merged**